### PR TITLE
Add new ANF API version 2019-07-01

### PR DESCRIPTION
**Pull request by leonardbf**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by leonardbf** [comment]
The breaking changes step has failed. We have seen this intermittently before even though there has been no change to the identified area. We recall that last time perhaps some other job was not running?

File: specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-07-01/netapp.json [hunk 1]
```diff
@@ -0,0 +1,2149 @@
...
+    },
+    "MountTargetName": {
+      "name": "mountTargetName",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "description": "The name of the mount target",
+      "x-ms-parameter-location": "method"
+    },
+    "SnapshotName
```
Comment (nschonni) [inline specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-07-01/netapp.json]:
```suggestion
      "default": "2019-07-01"
```

**Comment by yaohaizh** [comment]
@leonardbf, could you make your pull review with several commits:  One commit with copy of previous swagger spec. Then, make commits with diff from previous version.  This will make the review focusing on the change part. Otherwise, it is hard to review which part. Thanks.

**Comment by leonardbf** [comment]
PR closed and replaced with https://github.com/Azure/azure-rest-api-specs/pull/7020 in order to show the changes from the previous API

**Pull request closed without merging**